test/index.test.ts

```diff
@@ -67,3 +81,18 @@ describe('Testing various RPM sqlite databases', async () => {
     });
   }
 });
+
+function formatRpmPackages(
```

Comment:
Maybe worth renaming as we already have a function in the parser with the same name, to avoid confusion?